BusinessLogic/ILootLogic.cs

```diff
@@ -13,7 +13,7 @@ public interface ILootLogic
         LootItemViewModel CreateNewLootApplication(int id, int playerId, int? currentPlayerId);
         Task<bool> ApplyForLoot(int lootId, int playerId, string comments, int? currentPlayerId, bool specialOffer, bool deleteRequest);
         Task<bool> SetLootRequestStatus(int playerId, int currentPlayerId, int campaignLootId, int status, int lootStatus, string comments, bool dropRequestorDown);
-        LootOverviewViewModel GetOverview(int[] display, int syndicateId);
+        LootOverviewViewModel GetOverview(int[] display, int tierSelect, int typeSelect, int syndicateId);
```

Comment:
[Nitpicking]

I wouldn't include "Select" in the names of the variables here and in the underlying class. The lower-level business logic functions should be independent of the UI (or lack thereof) - and <select> thing belongs to HTML. Thus I'd name those variables `itemTier` and `itemType`.

Reply:
Sounds good.